lib/flux/app-state.ts

```diff
@@ -396,47 +371,20 @@ export const actionMap = new ActionMap({
       },
     },
 
-    selectNote(state: AppState, { note, hasRemoteUpdate }) {
+    selectNote(state: AppState) {
```

Comment:
right here it's evident how instead of a Redux model we built global setters in the application. we're asking "what data does this change" instead of "what changes this data"